common-tasks/dispvm.md

```diff
@@ -11,74 +11,78 @@ redirect_from:
 Disposable VMs (DispVMs)
 ========================
 
-Background
-----------
+A Disposable VM (DispVM) is a lightweight VM that can be created quickly and will disappear when closed. 
+Disposable VMs are usually created in order to host a single application, like a viewer, editor, or web browser. 
+This means that you can safely work with untrusted files without risk of compromising your other VMs. 
+DispVMs can be created either directly from Dom0 or from within AppVMs. 
+Once a DispVM has been created it will appear in Qubes VM Manager with the name "dispX". 
+Changes made to a file opened in a Disposable VM are passed back to the originating VM. 
 
-A Disposable VM (DispVM) is a lightweight VM that can be created quickly and which will disappear when it is finished with. Usually a Disposable VM is created in order to host a single application, like a viewer or an editor. This means that you can safely work with files without risk of compromising any of your VMs. Changes made to a file opened in a disposable VM are passed back to the originating VM. See [this article](https://blog.invisiblethings.org/2010/06/01/disposable-vms.html) for more on why would one want to use a Disposable VM.
+See [this article](https://blog.invisiblethings.org/2010/06/01/disposable-vms.html) for more on why one would want to use a Disposable VM.
 
-By default a DispVM will inherit the NetVM and firewall settings of the ancestor VM, that is the VM it is launched from. Thus if an AppVM uses sys-net as NetVM (instead of, say, sys-whonix), any DispVM launched from this AppVM will also have sys-net as its NetVM. You can change this behaviour for individual VMs: in Qubes VM Manager open VM Settings for the VM in question and go to the "Advanced" tab. Here you can edit the "NetVM for DispVM" setting to change the NetVM of any DispVM launched from that VM.
+Disposable VMs and Networking
+-----------------------------
 
-A DispVM launched from the Start Menu inherits the NetVM of the [DVM Template](https://www.qubes-os.org/doc/glossary/#dvm-template). By default it is named `fedora-XX-dvm` (where `XX` is the Fedora version of the default TemplateVM) and, as a so-called internal VM, hidden in Qubes VM Manager; it can be shown by selecting "Show/Hide internal VMs". Notice that changing the "NetVM for DispVM" setting for the DVM Template does *not* affect the NetVM of DispVMs launched from the Start Menu; only changing the DVM Template's own NetVM does.
```

Comment:
Why delete this without presenting the same information elsewhere? This is important.

Reply:
Ah, sorry, will reinsert.